### PR TITLE
Explicitly define SQS dependencies

### PR DESCRIPTION
**Pull request by hannseman**
Fixes: https://github.com/celery/celery/issues/7070

### CODE REVIEW

**Review by auvipy — CHANGES_REQUESTED** [review]
you can  check this as well https://github.com/celery/celery/pull/7069/files

**Comment by hannseman** [comment]
@auvipy not really sure what you mean but looks like you pulled the changes from this PR to yours so I guess this one can be closed 🤷

**Comment by auvipy** [comment]
to check if windows build actually passes with this change, which is still failing, can you check that branch?

**Comment by hannseman** [comment]
> to check if windows build actually passes with this change, which is still failing, can you check that branch?

Ah I see.

File: requirements/extras/sqs.txt [hunk 1]
```diff
@@ -1 +1,3 @@
-kombu[sqs]
+boto3>=1.9.12
+pycurl==7.43.0.2
+urllib3>=1.26.7 # Unittests are failing with urllib3>=1.28
```
Comment (hannseman) [inline requirements/extras/sqs.txt:3]: Not sure about this urllib3 dependency. I grabbed it from https://github.com/celery/kombu/commit/f3b04558fa0df4ecc383c93e0b3b300d95e17c47

The comment about version 1.28 and above breaking no longer makes much sense but from what I can see no 1.28 version exists for urllib3, see https://pypi.org/project/urllib3/#history 🤔

File: requirements/extras/sqs.txt [hunk 2]
```diff
@@ -1 +1,3 @@
-kombu[sqs]
+boto3>=1.9.12
+pycurl~=7.44.1
```
Comment (hannseman) [inline requirements/extras/sqs.txt:2]: Note that this version do not have any wheels. Latest version with wheels is [7.43.0.5](https://pypi.org/project/pycurl/7.43.0.5/#files)

**Comment by auvipy** [comment]
> you can check this as well https://github.com/celery/celery/pull/7069/files

closing in favor of it, as it cover more broader change. thanks for you time reporting and investigation.

**Pull request closed without merging**